pkg/internal/checkup/executor/executor.go

```diff
@@ -80,13 +80,16 @@ func (e Executor) Execute(ctx context.Context, vmiName, podName, podContainerNam
 
 	trexClient := trex.NewClient(e.podClient, e.namespace, podName, podContainerName, e.verbosePrintsEnabled)
 
+	testpmdConsole := NewTestpmdConsole(e.vmiSerialClient, e.namespace, e.vmiEastNICPCIAddress, e.vmiEastEthPeerMACAddress,
+		e.vmiWestNICPCIAddress, e.vmiWestEthPeerMACAddress, e.verbosePrintsEnabled)
```

Comment:
We need to see how do we reduce the number of inputs this function has.
It could be done in a follow-up PR.

Reply:
totally